cg/constants/orderforms.py

```diff
@@ -13,6 +13,8 @@
     str(OrderType.MIP_RNA),
 ]
 
+ORDERFORM_VERSIONS = {"1508": "27", "1604": "15", "1605": "10", "1603": "11", "2184": "7"}
```

Comment:
```suggestion
ORDERFORM_VERSIONS = { Orderform.BALSAMIC: "27", Orderform.FLUFFY: "15", Orderform.METAGENOME: "10", Orderform.Microsalt: "11", Orderform.SARS_COV_2: "7"}
```
It good to keep the connection and this is more readable for those of us who do not know the orderforrms by heart

Reply:
Fixed